colorama/ansitowin32.py

```diff
@@ -80,12 +88,41 @@ def __init__(self, wrapped, convert=None, strip=None, autoreset=False):
         # create the proxy wrapping our output stream
         self.stream = StreamWrapper(wrapped, self)
 
+        conversion_supported = True
         on_windows = os.name == 'nt'
         # We test if the WinAPI works, because even if we are on Windows
         # we may be using a terminal that doesn't support the WinAPI
         # (e.g. Cygwin Terminal). In this case it's up to the terminal
         # to support the ANSI codes.
-        conversion_supported = on_windows and winapi_test()
+        if on_windows and platform.release().startswith('10'):
```

Comment:
The comment above this line mismatches the code now. We do need to call `winapi_test` before using the other WinAPI functions inside the "if".